### ISSUE TITLE
Deep MNIST Tutorial would benefit from some diagrams of the neural network architecture 

### ISSUE THREAD

**Issue by hnarayanan** [tensorflow/tensorflow#3044]
The basic version of this tutorial has a [nice diagram](https://www.tensorflow.org/versions/r0.9/images/softmax-regression-scalargraph.png) that makes it really clear what model the code is attempting to reproduce.

The _Deep MNIST for Experts_ tutorial however doesn't have such an image, especially for the Multilayer Convolutional Network that is the heart of the article. I had a hard time (and still do, a bit) visualising what exactly the model code is attempting to do. I think the tutorial would benefit from some diagrams.

I am willing to contribute this if someone could handhold me a bit as to how the graphic above was generated and what the bar is for contributing to the docs.

**Comment by drpngx** [comment]
I think that's reasonable to have. Just submit a PR, and we'll handle it from there. Images are a bit tricky for us to sync, so we might remove them from github and push them separately to the documentation site. Feel free to open a PR and CC me on it.
